src/main/java/gregtech/common/tools/ToolUtility.java

```diff
@@ -105,7 +105,7 @@ public static void applyHammerDrops(Random random, IBlockState blockState, List<
             inputStack = new ItemStack(blockState.getBlock(), 1, blockState.getBlock().getMetaFromState(blockState));
         }
         MaterialStack input = OreDictUnifier.getMaterial(inputStack);
-        if (input != null && input.material.hasProperty(PropertyKey.ORE) && GTUtility.isOre(blockState.getBlock()) && !(player instanceof FakePlayer)) {
+        if (input != null && input.material.hasProperty(PropertyKey.ORE) && GTUtility.isOre(GTUtility.toItem(blockState)) && !(player instanceof FakePlayer)) {
```

Comment:
The `toItem` change can also be made on the following line (110) for cleanliness.